fe1-web/src/core/components/DropdownSelector.tsx

```diff
@@ -13,26 +13,35 @@ import containerStyles from '../styles/stylesheets/containerStyles';
  *  - Default selected option (not required, if not specified then first in the array is chosen)
  */
 
-const DropdownSelector = (props: IPropTypes) => {
-  const { selected } = props;
-  const { onChange } = props;
-  const { values } = props;
-  const options: any = [];
-  values.forEach((value) => options.push(<Picker.Item key={value} label={value || ''} />));
+const DropdownSelector = (
+  props: Omit<IPropTypes, 'onChange'> & {
+    // make the type for 'onChange' more concrete than the inferred type
+    onChange: (itemValue: string | null, itemIndex: number) => void;
+  },
```

Comment:
I'm wondering if casting `onChange` wouldn't be easier to read than this.
At the end of the day, neither options _really_ checks what is being passed beyond what `PropTypes` already does.

Reply:
That seems an oddly IDE-specific behaviour and I'd take code clarity over IDE happiness 😅
Which IDE are you using ? Are you redefining a new `const` variable taking the cast property ?

Reply:
I mean I don't think any IDE should be able to find out the correct type? The PropTypes just specify a generic function?

Reply:
What I meant is that the IDE does not know the type when doing

```tsx
<DropdownSelector onChange={...} />
```

Within the function it is fine using a cast